com.unity.visualeffectgraph/Editor/Compiler/VFXGraphCompiledData.cs

```diff
@@ -22,6 +22,7 @@ struct VFXContextCompiledData
         public VFXUniformMapper uniformMapper;
         public VFXMapping[] parameters;
         public int indexInShaderSource;
+        public int indexInSystemDesc;
```

Comment:
This does not seems needed. data index in systemDescs will always match order in compiledData

Reply:
See https://github.com/Unity-Technologies/Graphics/pull/4457#issuecomment-839768246